apps/web/src/common/feature-flags.js

```diff
@@ -0,0 +1,25 @@
+import config from '@pins/web/environment/config.js';
```

Comment:
what's the reason for the new "common" folder? is it something that could affect client-side code?

Reply:
yes - you might want to put client side code behind a flag